ops/utils/gauntlet.go

```
@@ -104,7 +104,7 @@ func (g Gauntlet) ReadCommandReport() (Report, error) {
 	}
 
 	var report Report
-	byteValue, _ := ioutil.ReadAll(jsonFile)
+	byteValue, _ := io.ReadAll(jsonFile)
```

But it doesn't complain about the ignored errors?